.eslintrc.json

```diff
@@ -125,18 +126,19 @@
       "files": ["*.ts"],
       "parser": "@typescript-eslint/parser",
       "plugins": [
-        "@typescript-eslint",
-        "eslint-plugin-tsdoc"
+        "@typescript-eslint"
       ],
       "extends": ["airbnb-typescript/lib/shared"],
       "rules": {
         "@typescript-eslint/no-implied-eval": "off",
         "@typescript-eslint/no-throw-literal": "off",
         "comma-dangle": ["warn", "never"],
+        "import/first": "off",
+        "import/prefer-default-export": "off",
         "lines-between-class-members": "off",
+        "lodash/prefer-lodash-typecheck": "off",
         "node/no-unsupported-features/es-syntax": "off",
-        "valid-jsdoc": "off",
-        "tsdoc/syntax": "warn"
+        "unicorn/no-abusive-eslint-disable": "off"
```

Comment:
You don't need to do this if you upgrade `eslint-plugin-unicorn` to the latest version. I just tested it. This issue proves that the plugin supports `@typescript` rules:

https://github.com/sindresorhus/eslint-plugin-unicorn/pull/212

Reply:
Fixed in 2a86d8408

Upgrading `eslint-plugin-unicorn` resulted 1,000+ new linter issues being reported. I disabled those rules in the config but have documented them so that we can discuss them in an architecture meeting and then file tickets to re-enable them.